genai-perf/genai_perf/subcommand/common.py

```diff
@@ -121,6 +118,11 @@ def get_extra_inputs_as_dict(args: Namespace) -> Dict[str, Any]:
                         f"Input name already exists in request_inputs dictionary: {input_name}"
                     )
                 request_inputs[input_name] = value
+            else:
+                raise ValueError(
+                    f"Invalid input format for --extra-inputs: {input_str}\n"
+                    "Expected input format: 'input_name:value'"
```

Comment:
Can we also update the error message to reflect the fact that no value (or flag) extra input is one of the expected format?

Reply:
Good idea, done!